src/test/resources/TestNG.xml

```diff
@@ -6,9 +6,9 @@
 
     <test name="Example Test" preserve-order="false">
         <classes>
+            <class name="BaseTest"/>
             <class name="Homework17" />
```

Comment:
Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://user-images.githubusercontent.com/104761482/217459691-4e10f32f-2ec1-4489-b9b8-7077144438d5.png)